src/20231122_AaveV3Bnb_AaveV3BNBActivation/AaveV3Bnb_AaveV3BNBActivation_20231122.sol

```diff
@@ -0,0 +1,240 @@
+    _supply(AaveV3Bnb.POOL, BTCB, BTCB_SEED_AMOUNT, address(AaveV3Bnb.COLLECTOR));
+    _supply(AaveV3Bnb.POOL, ETH, ETH_SEED_AMOUNT, address(AaveV3Bnb.COLLECTOR));
+    _supply(AaveV3Bnb.POOL, USDC, USDC_SEED_AMOUNT, address(AaveV3Bnb.COLLECTOR));
+    _supply(AaveV3Bnb.POOL, USDT, USDT_SEED_AMOUNT, address(AaveV3Bnb.COLLECTOR));
+  }
+
+  function newListings() public pure override returns (IAaveV3ConfigEngine.Listing[] memory) {
+    IAaveV3ConfigEngine.Listing[] memory listings = new IAaveV3ConfigEngine.Listing[](6);
+
+    listings[0] = IAaveV3ConfigEngine.Listing({
+      asset: CAKE,
+      assetSymbol: 'CAKE',
+      priceFeed: 0xB6064eD41d4f67e353768aA239cA86f4F73665a1,
+      eModeCategory: 0,
+      enabledToBorrow: EngineFlags.ENABLED,
+      stableRateModeEnabled: EngineFlags.DISABLED,
+      borrowableInIsolation: EngineFlags.DISABLED,
+      withSiloedBorrowing: EngineFlags.DISABLED,
+      flashloanable: EngineFlags.ENABLED,
+      ltv: 55_00,
+      liqThreshold: 61_00,
+      liqBonus: 10_00,
+      reserveFactor: 20_00,
+      supplyCap: 5_000_000,
+      borrowCap: 1_800_000,
+      debtCeiling: 7_000_000,
+      liqProtocolFee: 10_00,
+      rateStrategyParams: IV3RateStrategyFactory.RateStrategyParams({
+        optimalUsageRatio: _bpsToRay(45_00),
+        baseVariableBorrowRate: _bpsToRay(0),
+        variableRateSlope1: _bpsToRay(7_00),
+        variableRateSlope2: _bpsToRay(300_00),
+        stableRateSlope1: _bpsToRay(7_00),
+        stableRateSlope2: _bpsToRay(300_00),
+        baseStableRateOffset: _bpsToRay(2_00),
+        stableRateExcessOffset: _bpsToRay(8_00),
+        optimalStableToTotalDebtRatio: _bpsToRay(20_00)
+      })
+    });
+    listings[1] = IAaveV3ConfigEngine.Listing({
+      asset: WBNB,
+      assetSymbol: 'WBNB',
+      priceFeed: 0x0567F2323251f0Aab15c8dFb1967E4e8A7D42aeE,
+      eModeCategory: 0,
+      enabledToBorrow: EngineFlags.ENABLED,
+      stableRateModeEnabled: EngineFlags.DISABLED,
+      borrowableInIsolation: EngineFlags.DISABLED,
+      withSiloedBorrowing: EngineFlags.DISABLED,
+      flashloanable: EngineFlags.ENABLED,
+      ltv: 70_00,
+      liqThreshold: 75_00,
+      liqBonus: 10_00,
+      reserveFactor: 20_00,
+      supplyCap: 46_000,
+      borrowCap: 42_000,
+      debtCeiling: 0,
+      liqProtocolFee: 10_00,
+      rateStrategyParams: IV3RateStrategyFactory.RateStrategyParams({
+        optimalUsageRatio: _bpsToRay(45_00),
+        baseVariableBorrowRate: _bpsToRay(0),
+        variableRateSlope1: _bpsToRay(7_00),
+        variableRateSlope2: _bpsToRay(300_00),
+        stableRateSlope1: _bpsToRay(7_00),
+        stableRateSlope2: _bpsToRay(300_00),
+        baseStableRateOffset: _bpsToRay(2_00),
+        stableRateExcessOffset: _bpsToRay(8_00),
+        optimalStableToTotalDebtRatio: _bpsToRay(20_00)
+      })
+    });
+    listings[2] = IAaveV3ConfigEngine.Listing({
+      asset: BTCB,
+      assetSymbol: 'BTCB',
```

Comment:
Is this way or `BTCb`?

Reply:
The token symbol on the contract is `BTCB` but maybe `BTCb` is a bit easier to read